ui/Screen/DesignSystemGuide.svelte

```diff
@@ -84,20 +85,57 @@
     title="Typography"
     subTitle="Using GT America and GT America Mono from Grill Type">
```

Comment:
This should be changed to reflect the new font family names 😉

Reply:
Good catch ⚡️